Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:10-alpine
+FROM node:8-alpine
```

Comment:
How come this?

Reply:
it was always broken in 10 i only just spotted it here. I'll fix the 10 issues in an upcoming PR